backend/src/models/reports/Tab_8_rpt_CA_ContractsbySubcontractor.js

```diff
@@ -0,0 +1,128 @@
+// import knex database connection and utilities
+const { knex } = require("@database/databaseConnection")();
+const log = require("../../facilities/logging")(module.filename);
```

Comment:
this shouldn't be necessary if you throw the error back to the controller where it will be caught - see https://github.com/bcgov/gdx-agreements-tracker/blob/development/backend/src/models/reports/Tab_7_rpt_CA_ContractsbyContractor.js#L190